controllers/placementrule/placementrule_controller.go

```diff
@@ -585,3 +587,36 @@ func (r *PlacementRuleReconciler) SetupWithManager(mgr ctrl.Manager) error {
 	// create and return a new controller
 	return ctrBuilder.Complete(r)
 }
+
+func StartPlacementController(mgr manager.Manager) error {
+	if isplacementControllerRunnning {
+		return nil
+	}
+	isplacementControllerRunnning = true
```

Comment:
Do you think we need to cover this case? create CR --> start placement controller --> delete CR --> stop placement controller ---> create CR again --> start placement controller.

Reply:
@clyang82 
imo, it's not necessary. placement controller will do nothing if no mco cr.